src/destructors.md

```diff
@@ -325,6 +325,18 @@ let x = &mut 0;
 println!("{}", x);
 ```
 
+r[destructors.scope.lifetime-extension.const]
```

Comment:
I'm not sure if I am supposed to just come up with some name for this paragraph or what...

Cc @chorman0773

Reply:
Yeah, we're adding paragraph identifiers to the reference. If the file already has them, adding them in is good. Otherwise, you don't need to and they can be taken care of in the respective file PR.